### PR TITLE
Fix BACKUP_CUSTOM_LABEL

### PR DESCRIPTION
**Pull request by jan-brinkmann**
This fixes a bug concerning label BACKUP_CUSTOM_LABEL.

At the current state, the label has no effect. Even when it is set, the script stops and starts the container that have the non-custom label docker-volume-backup.stop-during-backup set as true. Now, the script stops and starts the container with the label defined in BACKUP_CUSTOM_LABEL set to true.

In the following example, the backup container stops and starts the data container as the label defined in BACKUP_CUSTOM_LABEL is set to true.

_backup container_
environment:
  - BACKUP_CUSTOM_LABEL: docker-volume-backup.stop-during-backup-custom

_data container_
labels:
  - "docker-volume-backup.stop-during-backup-custom=true"

### CODE REVIEW

**Comment by varhub** [comment]
Hi @jan-brinkmann, I guess you missed the point. Custom label is about create cohorts. So **all** commands apply only to a labeled subset.
In any case, I cannot ensure absence of typos/bugs at `--filter`.

TLDR: you should define 2 labels

**Comment by jan-brinkmann** [comment]
Hi @varhub, thanks for the response!
Can you please provide an example on how to work with BACKUP_CUSTOM_LABEL?
In my pull request, the containers are stoped and started as already explained, and commands defined in docker-volume-backup.exec-pre-backup and docker-volume-backup.exec-post-backup are applied in the associated containers.

**Comment by varhub** [comment]
I'm at mobile now, so I'll be brief.

Imagine that you duplicate your docker compose (at folders c1,c2). Now, c1_backup will see and stop both c1_db and c2_db.
Due to docker ps has not micro-managing nor scoped-access, scope is done via additional labeling to all containers of a swarn/compose/cohort.

Examples: BACKUP_CUSTOM_LABEL=graphana_hassio
BACKUP_CUSTOM_LABEL=cohort=3

Recommendation: create as many labels as you want and run `docker ps --filter` by hand

**Comment by jan-brinkmann** [comment]
Thank you for the explanation, @varhub!

After some more local experiments, I now have a better understanding on how it works.
